elements/pfe-content-set/demo/index.html

```diff
@@ -95,9 +95,12 @@ <h4 pfe-content-set--header>Heading 1</h4>
           varius blandit sit amet non magna. Etiam porta sem malesuada magna mollis euismod.</p>
       </div>
       <h4 pfe-content-set--header>Heading 2</h4>
-      <div pfe-content-set--panel>Content for heading 2. Sed posuere consectetur est at lobortis. Donec id elit non mi
+      <div pfe-content-set--panel>
```

Comment:
I don't think this is a breaking change but I wanted to double check.  Do `pfe-content-set` panels rely on its children to be wrapped in an element?  Can they still be text nodes?

Reply:
They can be either. I added this to the demo page to confirm incoming styles were retained.

Reply:
(Oh and this is just the demo page)